CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 - Support `fetch_schema` parameter for a connection (#219).
+- Corrected misspelling in the error code returned.
```

Comment:
1. I think it belongs to the ### Fixed section. (It's a mistake that another entry is outside the section, sorry for the confusing.)
2. Let's better talk about the effect in the changelog message, like [Fixed] "Error code on socket error.".

Reply:
Done

Reply:
Thank you! Can you squash the commits and force-push to the branch?

Reply:
Done